test/kb_ea_utils_server_test.py

```diff
@@ -473,7 +554,9 @@ def getPairedEndLib_SetInfo(self, read_libs_basename_list, refresh=False):
 
     ### TESTS 0.1-0.4: get Fastq Stats
     #
-    def test_get_fastq_ea_utils_stats(self):
+    @patch('biokbase.workspace.client.Workspace.get_objects',
+           side_effect=AssertionError('\n\n*****  Replace get_objects() with get_objects2()'))
+    def test_get_fastq_ea_utils_stats(self, mock):
```

Comment:
> it succeeds in local testing but not in the narrative

Wait, ea utils fails when run via the narrative but passes in tests? That seems like a big problem. Is there a ticket describing the symptoms and how to reproduce the problem?

Or are you saying that get_objects() fails when run in the narrative?